styles.css

```diff
@@ -0,0 +1,93 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100vw;
+  height: 100vh;
+  background-image: url('/images/image_geometry_1.svg');
+  background-repeat: no-repeat;
+}
+
+.hamburger {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.btn-icon {
+  background-image: url(/images/ic_menu.svg);
+  background-repeat: no-repeat;
+  width: 20px;
+  height: 20px;
+  border: none;
+  cursor: pointer;
+  background-color: transparent;
+  margin: 19px 16px 0 0;
+}
+
+.container {
+  margin-block-start: 104px;
+  padding-inline: 16px;
+}
+
+.name {
+  color: #1a2236;
+  font-family: 'Playfair Display', serif;
+  font-size: 50px;
+  line-height: 100%;
+  margin-block-end: 20px;
+  font-weight: 400;
+}
+
+.description {
+  color: #1a2236;
+  font-family: 'Poppins', sans-serif;
+  font-size: 14px;
+  line-height: 21px;
+  font-weight: 400;
+  letter-spacing: -0.01em;
+  margin-block-end: 40px;
+}
+
+.button {
+  margin-block-end: 32px;
+}
+
+.actual-button {
+  background: #1a2236;
+  color: #fff;
+  width: 100%;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  height: 50px;
+  text-align: center;
+}
+
+.bottom-image {
+  background-image: url(/images/image_geometry_2.svg);
```

Comment:
- [ ] This image is not being shown either. You could fix it by adding the `.` before the `/` like in the other cases. It should look like `background-image: url(./images/image_geometry_2.svg);` :smiley:

  ![image](https://user-images.githubusercontent.com/81522435/138363436-1a39d0c7-356f-4da0-a4f3-f31fe45a542d.png)